dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/documents/BaseDocumentReference.kt

```diff
@@ -10,5 +11,6 @@ import org.dataland.datalandbackend.interfaces.documents.BaseDocumentReference
 data class BaseDocumentReference(
     override val fileName: String? = null,
     @field:JsonProperty(required = true)
+    @field:NotBlank
```

Comment:
Do we need the required = true and the NotBlank annotation? 
Do i understand it correctly that the required= true annotation prevents the field from being not filled. But an empty string would be accepted. 
The NotBlank annotation no additionally prevents the backend from accepting an empty string. Is my assumption correct?

Reply:
ok then i resolve this here